### ISSUE TITLE
problem not loading css after installing snap theme on moodle 3.3

### ISSUE THREAD

**Issue by isabellaamal** [moodlerooms/moodle-theme_snap#108]
Hello, 
I have a Moodle 3.3 and I tried to change the theme to Snap (both from the Moodle site and from here) by copying the folder on the themes folder. When I change the theme it leaves the website without any theme and seems not to load CSS. 
I did clear the cache to no avail. 
I really like the snap theme and I would like to use it for the platform.
Is there a solution?
Please help,
Isabella

<img width="1440" alt="schermata 2018-02-15 alle 13 36 13" src="https://user-images.githubusercontent.com/36508131/36260882-feaf195c-1262-11e8-9f33-1f709de3c7a6.png">

**Comment by isabellaamal** [comment]
Still cannot solve the problem, any ideas?

**Comment by FinalRule** [comment]
I think if you updated your Moodle to 3.4 it will be fixed ... especially if you used the last version of snap form here not from Moodle plugins directory

**Comment by thauska** [comment]
Have you use the right branch?

**Comment by jpidesie** [comment]
Yes, I have the MOODLE_33_STABLE branch and Moodle 3.3.7. I deleted the class js-snap-pm-trigger in moodle/theme/snap/classes/output/core_renderer.php The login modal not show :( but users go directly to login page
line 526: 'class' => 'btn btn-default snap-login-button', //jpidesie delete class js-snap-pm-trigger
